Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -671,7 +671,8 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
         var impartedSettings = PIF.BuildSettings()
 
         var cFlags: [String] = []
-        if let result = pkgConfigArgs(for: systemTarget, diagnostics: diagnostics, fileSystem: fileSystem) {
+        for resultw in pkgConfigArgs(for: systemTarget, diagnostics: diagnostics, fileSystem: fileSystem) {
+            guard let result = resultw else { continue }
```

Comment:
This would be cleaned up if the array were to contain only the results that were found.